core/src/main/java/hudson/util/MaskingClassLoader.java

```diff
@@ -62,7 +72,20 @@ public MaskingClassLoader(ClassLoader parent, Collection<String> masks) {
         return super.loadClass(name, resolve);
     }
 
+    @Override
+    public synchronized URL getResource(String name) {
```

Comment:
Thanks for the merge, where should I add the fix for the `getResources` ? Is it in a new pull request?